packages/components/src/components/wizard/wizard.jsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import Header from './wizard-header.jsx';
+import Icon from '../icon/icon.jsx';
+
+const Wizard = React.forwardRef(({ children, steps, lbl_previous, has_prev, lbl_next, has_next, className }, ref) => {
+    const [step, setStep] = React.useState(0);
+    const [component, setComponent] = React.useState(steps[0].component);
+    const last_step = steps.length - 1;
+
+    const nextStep = () => {
+        const next_step = step + 1;
+        const is_last_step = next_step > last_step;
+        if (is_last_step) return;
+        setStep(next_step);
+        setComponent(steps[next_step].component);
+    };
+
+    const prevStep = () => {
+        const prev_step = step - 1;
+        const is_first_step = prev_step < 0;
+        if (is_first_step) return;
+        setStep(prev_step);
+        setComponent(steps[prev_step].component);
+    };
+
+    const selectStep = selected_step => {
+        const is_prev_step_not_allowed = selected_step < 0;
+        const is_next_step_not_allowed = selected_step > last_step;
+        if (is_prev_step_not_allowed || is_next_step_not_allowed) return;
+        setStep(selected_step);
+        setComponent(steps[selected_step].component);
+    };
+
+    React.useImperativeHandle(ref, () => ({
+        prevStep,
+        nextStep,
+        selectStep,
+    }));
+
+    const prevButton = () =>
+        step === 0 ? (
+            <div className='wizard__placeholder' />
+        ) : (
+            <a onClick={prevStep} className='wizard__btn wizard__btn-left'>
```

Comment:
I think this should be a `<button />` since we're not navigating the user away from the current page

Reply:
Current `Button` component does not support icon alignment.

Reply:
I mea the html button element not the React component @matin-binary